conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptionHandlerTest.java

```diff
@@ -90,6 +96,48 @@ public void handlesRemoteException() throws IOException {
         assertThat(response.code()).isEqualTo(ErrorType.INTERNAL.httpErrorCode());
     }
 
+    @Test
+    public void handlesQosExceptionThrottleWithoutDuration() throws IOException {
```

Comment:
These methods match the old ones in https://github.com/palantir/conjure-java-runtime/blob/develop/conjure-java-jersey-server/src/test/java/com/palantir/conjure/java/server/jersey/QosExceptionMapperTest.java as much as possible